test/testUtils.cpp

```diff
@@ -86,9 +86,9 @@ void interpolateWachspressSphericalTest(MPMesh& mpMesh){
                                               MPsPosition(mp,0),
                                               MPsPosition(mp,1),
                                               MPsPosition(mp,2));*/
-            //  assert(wp_coord[0] - MPsPosition(mp,0) < TEST_EPSILON);
-            // assert(wp_coord[1] - MPsPosition(mp,1) < TEST_EPSILON);
-            // assert(wp_coord[2] - MPsPosition(mp,2) < TEST_EPSILON);
+            //assert(wp_coord[0] - MPsPosition(mp,0) < TEST_EPSILON);
+            //assert(wp_coord[1] - MPsPosition(mp,1) < TEST_EPSILON);
+            //assert(wp_coord[2] - MPsPosition(mp,2) < TEST_EPSILON);
```

Comment:
Please remove these lines.  We don't want commented code in the repo.  It will just cause confusion in the future.

Reply:
As far as I can tell these commented lines of core are removed.